### PR TITLE
qFlipper: 0.8.2 -> 1.0.1

### PR DESCRIPTION
**Pull request by cab404**
###### Description of changes

qFlipper package bump, and related build fixes

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [X] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/qflipper/default.nix [hunk 1]
```diff
@@ -82,20 +87,13 @@ mkDerivation {
   installPhase = ''
     runHook preInstall
 
+    make install
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/qflipper/default.nix:90]: This is not how it should be done. If you don't overwrite installPhase nix will automatically call make with the right arguments. The other stuff here should then be moved to postInstall.

File: pkgs/tools/misc/qflipper/default.nix [hunk 2]
```diff
@@ -70,6 +70,11 @@ mkDerivation {
     qtwayland
   ];
 
+  qmakeFlags = [
+    "DEFINES+=DISABLE_APPLICATION_UPDATES"
+    "CONFIG+=qtquickcompiler"
+  ];
+
   preBuild = ''
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/qflipper/default.nix:78]:
```suggestion
  postPatch = ''
```

File: pkgs/tools/misc/qflipper/default.nix [hunk 3]
```diff
@@ -21,10 +21,10 @@
...
 let
-  version = "0.8.2";
+  version = "1.0.1";
   timestamp = "99999999999";
   commit = "nix-${version}";
-  hash = "sha256-BaqKlF2SZueykFhtj91McP39oXYAx+lz8eXhn5eouqg=";
+  hash = "sha256-vHBlrtQ06kjjXXGL/jSdpAPHgqb7Vn1c6jXZVXwxHPQ=";
 
   udev_rules = ''
     #Flipper Zero serial port
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/qflipper/default.nix:30]: wrapQtAppsHook goes into nativeBuildInputs.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by cab404** [comment]
ty, @SuperSandro2000

**Pull request merged**